static/scripts/rewards/web3/invalidate-nonce.ts

```diff
@@ -6,5 +6,9 @@ import { nonceBitmap } from "./nonce-bitmap";
 export async function invalidateNonce(signer: ethers.providers.JsonRpcSigner, nonce: BigNumber): Promise<void> {
   const permit2Contract = new ethers.Contract(PERMIT2_ADDRESS, permit2Abi, signer);
   const { wordPos, bitPos } = nonceBitmap(nonce);
-  await permit2Contract.invalidateUnorderedNonces(wordPos, bitPos);
+  // mimics https://github.com/ubiquity/pay.ubq.fi/blob/c9e7ed90718fe977fd9f348db27adf31d91d07fb/scripts/solidity/test/Permit2.t.sol#L428
```

Comment:
It's not easy for me to copy paste from the iOS client I'll check later on my computer.